.gitignore

```diff
@@ -0,0 +1,3 @@
+py_compile.sh
```

Comment:
Are these generated files? If not, you probably want to put them in your personal, global gitignore, rather than this project-specific one.

Reply:
Yes my mistake.

Reply:
This file has been removed.